tests/reftests/pin.test

```diff
@@ -1,7 +1,811 @@
+  - install nip-path 42 (pinned)
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+-> retrieved nip-path.42  (file://${BASEDIR}/nip-path)
+-> installed nip-path.42
+Done.
+### : pin version with version
+### opam pin add nip-v 1 --with-version 2 --no-action
+[nip-v.2] synchronised (file://${BASEDIR}/nip-v.1.tgz)
+nip-v is now pinned to file://${BASEDIR}/nip-v.1.tgz (version 2)
+### opam pin add nip 1 --with-version 42 --no-action
+[nip.42] synchronised (file://${BASEDIR}/nip.1.tgz)
+[NOTE] No package definition found for nip.42: please complete the template
+[WARNING] The opam file didn't pass validation:
+             error 22: Some fields are present but empty; remove or fill them: "homepage", "license", "bug_reports"
+             error 57: Synopsis must not be empty
+           warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/ : ""
+Proceed anyway ('no' will re-edit)? [y/n] y
+You can edit this file again with "opam pin edit nip", export it with "opam show nip --raw"
+nip is now pinned to file://${BASEDIR}/nip.1.tgz (version 42)
+### opam pin list
+nip.42       (uninstalled)  rsync  file://${BASEDIR}/nip.1.tgz
+nip-path.42                 rsync  file://${BASEDIR}/nip-path
+nip-v.2      (uninstalled)  rsync  file://${BASEDIR}/nip-v.1.tgz
+### find OPAM/versions/lib/ -name '*.t' | sort
+OPAM/versions/lib/nip-path/nip-path.ved.t
+### :::::::::::::::::::::::
+### :VII: Linting
+### :::::::::::::::::::::::
+### <nip-empty/nip-empty.opam>
+opam-version: "2.0"
+### <nip-empty2/nip-empty2.opam>
+opam-version: "2.0"
+### opam pin add nip-empty ./nip-empty
+Package nip-empty does not exist, create as a NEW package? [y/n] y
+[nip-empty.dev] synchronised (file://${BASEDIR}/nip-empty)
+[WARNING] Failed checks on nip-empty package definition from source at file://${BASEDIR}/nip-empty:
+             error 23: Missing field 'maintainer'
+           warning 25: Missing field 'authors'
+           warning 35: Missing field 'homepage'
+           warning 36: Missing field 'bug-reports'
+             error 57: Synopsis must not be empty
+           warning 68: Missing field 'license'
+nip-empty is now pinned to file://${BASEDIR}/nip-empty (version dev)
+
+The following actions will be performed:
+=== install 1 package
+  - install nip-empty dev (pinned)
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+-> retrieved nip-empty.dev  (file://${BASEDIR}/nip-empty)
+-> installed nip-empty.dev
+Done.
+### opam pin add ./nip-empty2
+Package nip-empty2 does not exist, create as a NEW package? [y/n] y
+nip-empty2 is now pinned to file://${BASEDIR}/nip-empty2 (version dev)
+
+The following actions will be performed:
+=== install 1 package
+  - install nip-empty2 dev (pinned)
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+-> retrieved nip-empty2.dev  (file://${BASEDIR}/nip-empty2)
+-> installed nip-empty2.dev
+Done.
```

Comment:
Is it a bug?

Reply:
the fact that linting is not done if the name of the package is not specified? yeah probably although it doesn't seem critical